atomic_reactor/utils/cachito.py

```diff
@@ -67,6 +67,8 @@ def request_sources(self, repo, ref, flags=None, pkg_managers=None, user=None,
         :param user: str, user the request is created on behalf of. This is reserved for privileged
                      users that can act as cachito representatives
         :param dependency_replacements: list<dict>, dependencies to be replaced by cachito
+        :param packages: dict, the packages configuration that allows to specify things such
```

Comment:
On a second thought, is this a good name for this param, considering we will allow multiple packages to be requested in a single `Request`?

Reply:
In IRC I think we decided to keep the name as is.